precompiles/common/types.go

```diff
@@ -111,3 +112,22 @@ func SafeAdd(a, b sdkmath.Int) (res *big.Int, overflow bool) {
 	res = a.BigInt().Add(a.BigInt(), b.BigInt())
 	return res, res.BitLen() > sdkmath.MaxBitLen
 }
+
+// AccAddressFromBech32 creates an AccAddress from a Bech32 string.
+func AccAddressFromBech32(address string, bech32prefix string) (addr sdk.AccAddress, err error) {
```

Comment:
Please, move this function to `utils/utils.go` 🙏. AccAddressFromBech32 is an utility also outside precompiles. To distinguish it from the sdk version I would also change the name in CreateAccAddressFromBech32